packages/design/src/shadows.css

```diff
@@ -2,4 +2,5 @@
   --shadow-base: 0px 1px 4px 0px rgba(0, 0, 0, 0.0980392),
     0px 4px 12px 0px rgba(0, 0, 0, 0.0470588);
   --shadow-focus: 0px 0px 4px 2px var(--color-outline);
+  --shadow-dark: 0px 0px 4px 2px var(--color-blue--lightest);
```

Comment:
@whereyoubinns 2 comments on this shadow:

- let's move this into the Spinner.css file for now. If we need to use it in other cases it can be brought up here, but for now I think the spinner is probably a special case
- we don't have a nice systemic way of doing this with CSS variables, but can we use the RGBA values from `--color-blue` to make this shadow ~40% opacity?  `rgba(1, 41, 57, 0.4)` 
- It will give us a similar visual appearance on a white background, but act as a more "shadowy" shadow if it's overlaying content